src/components/OnboardingStepper.js

```diff
@@ -91,7 +184,18 @@ const OnboardingStepper = ({
           variant="progress"
         />
       </Header>
-      <View>
+      {todoRemoveFlag && current >= screenNames.ADD_PHOTO && (
+        <AvatarHeader hideImage={current == screenNames.ADD_PHOTO} />
+      )}
+      <View mt={8}>
```

Comment:
So it is different way of using margins paddings. Instead of using class you can use those values and they are multiply by a number from config which is 8. So it will give you 64px.